### ISSUE TITLE
Elasticsearch doesn't start after plugins are installed

### ISSUE THREAD

**Issue by ericofusco** [floragunncom/search-guard#201]
Search Guard 2.3.5.5
Search Guard SSL 2.3.5.15
ElasticSearch 2.3.5
OpenJDK 7u101
Ubuntu 14.04.1
2 Nodes

Error below happens when I try to start elasticsearch, no SG settings added. This happens after I install SG plugins and restart ElasticSearch.

If I remove plugin search guard ssl the same error occurs with search guard plugin, when I remove both plugins elasticsearch starts normally.

Exception: java.lang.IllegalStateException: Could not load plugin descriptor for existing plugin [search-guard-2]. Was the plugin built before 2.0?

Logs (DEBUG level)
```
[2016-08-18 19:17:46,274][DEBUG][bootstrap                ] Linux seccomp filter installation successful, threads: [all]
[2016-08-18 19:17:47,838][DEBUG][bootstrap                ] java.class.path: /usr/share/elasticsearch/lib/elasticsearch-2.3.5.jar:/usr/share/elasticsearch/lib/lucene-core-5.5.0.jar:/usr/share/elasticsearch/lib/s
[2016-08-18 19:17:47,838][DEBUG][bootstrap                ] sun.boot.class.path: /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/resources.jar:/usr/lib/jvm/java-7-openjdk-amd64/jre/lib/rt.jar:/usr/lib/jvm/java-7-openj
[2016-08-18 19:17:47,838][DEBUG][bootstrap                ] classloader urls: [file:/usr/share/elasticsearch/lib/elasticsearch-2.3.5.jar, file:/usr/share/elasticsearch/lib/lucene-core-5.5.0.jar, file:/usr/share/
[2016-08-18 19:17:47,844][DEBUG][bootstrap                ] java.home: /usr/lib/jvm/java-7-openjdk-amd64/jre
[2016-08-18 19:17:47,845][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/elasticsearch-2.3.5.jar
[2016-08-18 19:17:47,901][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-core-5.5.0.jar
[2016-08-18 19:17:47,915][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/snakeyaml-1.15.jar
[2016-08-18 19:17:47,918][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/netty-3.10.5.Final.jar
[2016-08-18 19:17:47,926][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jackson-dataformat-yaml-2.6.6.jar
[2016-08-18 19:17:47,931][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/spatial4j-0.5.jar
[2016-08-18 19:17:47,932][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jsr166e-1.1.0.jar
[2016-08-18 19:17:47,933][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/securesm-1.0.jar
[2016-08-18 19:17:47,933][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/commons-cli-1.3.1.jar
[2016-08-18 19:17:47,935][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jts-1.13.jar
[2016-08-18 19:17:47,941][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-misc-5.5.0.jar
[2016-08-18 19:17:47,943][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jackson-core-2.6.6.jar
[2016-08-18 19:17:47,945][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/HdrHistogram-2.1.6.jar
[2016-08-18 19:17:47,947][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-sandbox-5.5.0.jar
[2016-08-18 19:17:47,949][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/log4j-1.2.17.jar
[2016-08-18 19:17:47,953][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/compiler-0.8.13.jar
[2016-08-18 19:17:47,954][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/t-digest-3.0.jar
[2016-08-18 19:17:47,955][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-highlighter-5.5.0.jar
[2016-08-18 19:17:47,956][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-analyzers-common-5.5.0.jar
[2016-08-18 19:17:47,960][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/joda-convert-1.2.jar
[2016-08-18 19:17:47,961][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/joda-time-2.9.4.jar
[2016-08-18 19:17:47,965][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jackson-dataformat-cbor-2.6.6.jar
[2016-08-18 19:17:47,966][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-queries-5.5.0.jar
[2016-08-18 19:17:47,968][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-join-5.5.0.jar
[2016-08-18 19:17:47,969][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/apache-log4j-extras-1.2.17.jar
[2016-08-18 19:17:47,974][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/compress-lzf-1.0.2.jar
[2016-08-18 19:17:47,976][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-memory-5.5.0.jar
[2016-08-18 19:17:47,977][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/guava-18.0.jar
[2016-08-18 19:17:47,993][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-suggest-5.5.0.jar
[2016-08-18 19:17:47,995][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-grouping-5.5.0.jar
[2016-08-18 19:17:47,997][DEBUG][bootstrap                ] excluding duplicate classpath element: /usr/share/elasticsearch/lib/elasticsearch-2.3.5.jar
[2016-08-18 19:17:47,997][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jackson-dataformat-smile-2.6.6.jar
[2016-08-18 19:17:47,999][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-spatial-5.5.0.jar
[2016-08-18 19:17:48,001][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-backward-codecs-5.5.0.jar
[2016-08-18 19:17:48,004][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jna-4.1.0.jar
[2016-08-18 19:17:48,007][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-spatial3d-5.5.0.jar
[2016-08-18 19:17:48,008][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/hppc-0.7.1.jar
[2016-08-18 19:17:48,017][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-queryparser-5.5.0.jar
[2016-08-18 19:17:48,251][INFO ][node                     ] [node1] version[2.3.5], pid[4095], build[90f439f/2016-07-27T10:36:52Z]
[2016-08-18 19:17:48,251][INFO ][node                     ] [node1] initializing ...
[2016-08-18 19:17:48,252][DEBUG][node                     ] [node1] using config [/etc/elasticsearch], data [[/var/data/elasticsearch]], logs [/var/log/elasticsearch], plugins [/usr/share/elasticsearch/plugins]
[2016-08-18 19:17:48,274][DEBUG][bootstrap                ] java.home: /usr/lib/jvm/java-7-openjdk-amd64/jre
[2016-08-18 19:17:48,274][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/elasticsearch-2.3.5.jar
[2016-08-18 19:17:48,311][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-core-5.5.0.jar
[2016-08-18 19:17:48,314][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/snakeyaml-1.15.jar
[2016-08-18 19:17:48,323][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/netty-3.10.5.Final.jar
[2016-08-18 19:17:48,325][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jackson-dataformat-yaml-2.6.6.jar
[2016-08-18 19:17:48,326][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/spatial4j-0.5.jar
[2016-08-18 19:17:48,327][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jsr166e-1.1.0.jar
[2016-08-18 19:17:48,327][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/securesm-1.0.jar
[2016-08-18 19:17:48,328][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/commons-cli-1.3.1.jar
[2016-08-18 19:17:48,329][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jts-1.13.jar
[2016-08-18 19:17:48,330][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-misc-5.5.0.jar
[2016-08-18 19:17:48,339][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jackson-core-2.6.6.jar
[2016-08-18 19:17:48,340][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/HdrHistogram-2.1.6.jar
[2016-08-18 19:17:48,340][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-sandbox-5.5.0.jar
[2016-08-18 19:17:48,341][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/log4j-1.2.17.jar
[2016-08-18 19:17:48,342][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/compiler-0.8.13.jar
[2016-08-18 19:17:48,343][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/t-digest-3.0.jar
[2016-08-18 19:17:48,343][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-highlighter-5.5.0.jar
[2016-08-18 19:17:48,344][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-analyzers-common-5.5.0.jar
[2016-08-18 19:17:48,346][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/joda-convert-1.2.jar
[2016-08-18 19:17:48,346][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/joda-time-2.9.4.jar
[2016-08-18 19:17:48,356][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jackson-dataformat-cbor-2.6.6.jar
[2016-08-18 19:17:48,357][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-queries-5.5.0.jar
[2016-08-18 19:17:48,358][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-join-5.5.0.jar
[2016-08-18 19:17:48,359][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/apache-log4j-extras-1.2.17.jar
[2016-08-18 19:17:48,360][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/compress-lzf-1.0.2.jar
[2016-08-18 19:17:48,361][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-memory-5.5.0.jar
[2016-08-18 19:17:48,361][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/guava-18.0.jar
[2016-08-18 19:17:48,372][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-suggest-5.5.0.jar
[2016-08-18 19:17:48,373][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-grouping-5.5.0.jar
[2016-08-18 19:17:48,374][DEBUG][bootstrap                ] excluding duplicate classpath element: /usr/share/elasticsearch/lib/elasticsearch-2.3.5.jar
[2016-08-18 19:17:48,374][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jackson-dataformat-smile-2.6.6.jar
[2016-08-18 19:17:48,375][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-spatial-5.5.0.jar
[2016-08-18 19:17:48,376][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-backward-codecs-5.5.0.jar
[2016-08-18 19:17:48,377][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jna-4.1.0.jar
[2016-08-18 19:17:48,378][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-spatial3d-5.5.0.jar
[2016-08-18 19:17:48,387][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/hppc-0.7.1.jar
[2016-08-18 19:17:48,388][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-queryparser-5.5.0.jar
[2016-08-18 19:17:48,389][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/modules/lang-expression/asm-5.0.4.jar
[2016-08-18 19:17:48,390][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/modules/lang-expression/lucene-expressions-5.5.0.jar
[2016-08-18 19:17:48,391][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/modules/lang-expression/antlr4-runtime-4.5.1-1.jar
[2016-08-18 19:17:48,392][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/modules/lang-expression/lang-expression-2.3.5.jar
[2016-08-18 19:17:48,393][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/modules/lang-expression/asm-commons-5.0.4.jar
[2016-08-18 19:17:48,810][DEBUG][bootstrap                ] java.home: /usr/lib/jvm/java-7-openjdk-amd64/jre
[2016-08-18 19:17:48,810][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/elasticsearch-2.3.5.jar
[2016-08-18 19:17:48,820][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-core-5.5.0.jar
[2016-08-18 19:17:48,822][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/snakeyaml-1.15.jar
[2016-08-18 19:17:48,823][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/netty-3.10.5.Final.jar
[2016-08-18 19:17:48,825][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jackson-dataformat-yaml-2.6.6.jar
[2016-08-18 19:17:48,826][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/spatial4j-0.5.jar
[2016-08-18 19:17:48,827][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jsr166e-1.1.0.jar
[2016-08-18 19:17:48,828][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/securesm-1.0.jar
[2016-08-18 19:17:48,828][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/commons-cli-1.3.1.jar
[2016-08-18 19:17:48,829][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jts-1.13.jar
[2016-08-18 19:17:48,830][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-misc-5.5.0.jar
[2016-08-18 19:17:48,831][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jackson-core-2.6.6.jar
[2016-08-18 19:17:48,831][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/HdrHistogram-2.1.6.jar
[2016-08-18 19:17:48,832][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-sandbox-5.5.0.jar
[2016-08-18 19:17:48,833][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/log4j-1.2.17.jar
[2016-08-18 19:17:48,834][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/compiler-0.8.13.jar
[2016-08-18 19:17:48,834][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/t-digest-3.0.jar
[2016-08-18 19:17:48,835][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-highlighter-5.5.0.jar
[2016-08-18 19:17:48,835][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-analyzers-common-5.5.0.jar
[2016-08-18 19:17:48,837][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/joda-convert-1.2.jar
[2016-08-18 19:17:48,837][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/joda-time-2.9.4.jar
[2016-08-18 19:17:48,839][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jackson-dataformat-cbor-2.6.6.jar
[2016-08-18 19:17:48,839][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-queries-5.5.0.jar
[2016-08-18 19:17:48,840][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-join-5.5.0.jar
[2016-08-18 19:17:48,841][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/apache-log4j-extras-1.2.17.jar
[2016-08-18 19:17:48,842][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/compress-lzf-1.0.2.jar
[2016-08-18 19:17:48,843][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-memory-5.5.0.jar
[2016-08-18 19:17:48,843][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/guava-18.0.jar
[2016-08-18 19:17:48,846][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-suggest-5.5.0.jar
[2016-08-18 19:17:48,847][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-grouping-5.5.0.jar
[2016-08-18 19:17:48,848][DEBUG][bootstrap                ] excluding duplicate classpath element: /usr/share/elasticsearch/lib/elasticsearch-2.3.5.jar
[2016-08-18 19:17:48,848][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jackson-dataformat-smile-2.6.6.jar
[2016-08-18 19:17:48,848][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-spatial-5.5.0.jar
[2016-08-18 19:17:48,849][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-backward-codecs-5.5.0.jar
[2016-08-18 19:17:48,850][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jna-4.1.0.jar
[2016-08-18 19:17:48,850][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-spatial3d-5.5.0.jar
[2016-08-18 19:17:48,851][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/hppc-0.7.1.jar
[2016-08-18 19:17:48,852][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-queryparser-5.5.0.jar
[2016-08-18 19:17:48,852][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/modules/reindex/reindex-2.3.5.jar
[2016-08-18 19:17:48,871][DEBUG][bootstrap                ] java.home: /usr/lib/jvm/java-7-openjdk-amd64/jre
[2016-08-18 19:17:48,871][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/elasticsearch-2.3.5.jar
[2016-08-18 19:17:48,876][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-core-5.5.0.jar
[2016-08-18 19:17:48,878][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/snakeyaml-1.15.jar
[2016-08-18 19:17:48,878][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/netty-3.10.5.Final.jar
[2016-08-18 19:17:48,879][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jackson-dataformat-yaml-2.6.6.jar
[2016-08-18 19:17:48,880][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/spatial4j-0.5.jar
[2016-08-18 19:17:48,880][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jsr166e-1.1.0.jar
[2016-08-18 19:17:48,881][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/securesm-1.0.jar
[2016-08-18 19:17:48,881][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/commons-cli-1.3.1.jar
[2016-08-18 19:17:48,882][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jts-1.13.jar
[2016-08-18 19:17:48,883][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-misc-5.5.0.jar
[2016-08-18 19:17:48,884][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jackson-core-2.6.6.jar
[2016-08-18 19:17:48,885][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/HdrHistogram-2.1.6.jar
[2016-08-18 19:17:48,885][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-sandbox-5.5.0.jar
[2016-08-18 19:17:48,886][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/log4j-1.2.17.jar
[2016-08-18 19:17:48,892][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/compiler-0.8.13.jar
[2016-08-18 19:17:48,892][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/t-digest-3.0.jar
[2016-08-18 19:17:48,893][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-highlighter-5.5.0.jar
[2016-08-18 19:17:48,893][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-analyzers-common-5.5.0.jar
[2016-08-18 19:17:48,896][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/joda-convert-1.2.jar
[2016-08-18 19:17:48,896][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/joda-time-2.9.4.jar
[2016-08-18 19:17:48,898][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jackson-dataformat-cbor-2.6.6.jar
[2016-08-18 19:17:48,899][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-queries-5.5.0.jar
[2016-08-18 19:17:48,900][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-join-5.5.0.jar
[2016-08-18 19:17:48,900][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/apache-log4j-extras-1.2.17.jar
[2016-08-18 19:17:48,901][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/compress-lzf-1.0.2.jar
[2016-08-18 19:17:48,902][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-memory-5.5.0.jar
[2016-08-18 19:17:48,902][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/guava-18.0.jar
[2016-08-18 19:17:48,905][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-suggest-5.5.0.jar
[2016-08-18 19:17:48,906][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-grouping-5.5.0.jar
[2016-08-18 19:17:48,907][DEBUG][bootstrap                ] excluding duplicate classpath element: /usr/share/elasticsearch/lib/elasticsearch-2.3.5.jar
[2016-08-18 19:17:48,907][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jackson-dataformat-smile-2.6.6.jar
[2016-08-18 19:17:48,908][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-spatial-5.5.0.jar
[2016-08-18 19:17:48,908][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-backward-codecs-5.5.0.jar
[2016-08-18 19:17:48,909][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/jna-4.1.0.jar
[2016-08-18 19:17:48,910][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-spatial3d-5.5.0.jar
[2016-08-18 19:17:48,911][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/hppc-0.7.1.jar
[2016-08-18 19:17:48,912][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/lib/lucene-queryparser-5.5.0.jar
[2016-08-18 19:17:48,913][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/modules/lang-groovy/lang-groovy-2.3.5.jar
[2016-08-18 19:17:48,914][DEBUG][bootstrap                ] examining jar: /usr/share/elasticsearch/modules/lang-groovy/groovy-2.4.6-indy.jar
[2016-08-18 19:17:48,934][ERROR][bootstrap                ] Exception
java.lang.IllegalStateException: Could not load plugin descriptor for existing plugin [search-guard-2]. Was the plugin built before 2.0?
    at org.elasticsearch.plugins.PluginsService.getPluginBundles(PluginsService.java:381)
    at org.elasticsearch.plugins.PluginsService.<init>(PluginsService.java:128)
    at org.elasticsearch.node.Node.<init>(Node.java:158)
    at org.elasticsearch.node.Node.<init>(Node.java:140)
    at org.elasticsearch.node.NodeBuilder.build(NodeBuilder.java:143)
    at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:178)
    at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:270)
    at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:35)
Caused by: java.nio.file.AccessDeniedException: /usr/share/elasticsearch/plugins/search-guard-2/plugin-descriptor.properties
    at sun.nio.fs.UnixException.translateToIOException(UnixException.java:84)
    at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
    at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
    at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:214)
    at java.nio.file.Files.newByteChannel(Files.java:317)
    at java.nio.file.Files.newByteChannel(Files.java:363)
    at java.nio.file.spi.FileSystemProvider.newInputStream(FileSystemProvider.java:380)
    at java.nio.file.Files.newInputStream(Files.java:108)
    at org.elasticsearch.plugins.PluginInfo.readFromProperties(PluginInfo.java:87)
    at org.elasticsearch.plugins.PluginsService.getPluginBundles(PluginsService.java:378)
    ... 7 more                                                                                                                            
```

**Comment by floragunncom** [comment]
Have a look at the root cause of this Exception:

Caused by: java.nio.file.AccessDeniedException: /usr/share/elasticsearch/plugins/search-guard-2/plugin-descriptor.properties

This seems like a simple file permission problem. Most likely you run Elastic with one account, and have installed the plugin with another account.

**Comment by ericofusco** [comment]
Thank you @floragunncom, sorry about that. I should've noticed the cause line. I installed the plugins with the correct user and it fixed the issue.

I've installed ElasticSearch using their official debian packages that runs elasticsearch with elasticsearch user.

I suggest to mention that in the README file and documentation. Instructions say to install plugins with "sudo" command. I thought the plugins should be installed with root only.

Thanks again.
